tests/src/no_package_with_message.proto

```diff
@@ -0,0 +1,5 @@
+syntax = "proto3";
+
+message NoPackageWithMessageExampleMsg {
+  string some_field = 1;
+}
```

Comment:
I believe this test already exists: `no_root_packages`. Can you extend the test to also test your usecase?